e2e/utils.go

```diff
@@ -867,6 +871,29 @@ func validatePVCSnapshot(
 				LabelSelector: fmt.Sprintf("%s=%s", appKey, label[appKey]),
 			}
 			wgErrs[n] = createPVCAndApp(name, f, &p, &a, deployTimeout)
+			if wgErrs[n] == nil && restoreKMS != noKMS {
+				if restoreKMS.canGetPassphrase() {
+					imageData, sErr := getImageInfoFromPVC(p.Namespace, name, f)
+					if sErr != nil {
+						wgErrs[n] = fmt.Errorf(
+							"failed to get image info for %s namespace=%s volumehandle=%s error=%w",
+							name,
+							p.Namespace,
+							imageData.csiVolumeHandle,
+							sErr)
+					} else {
+						// check new passphrase created
+						stdOut, stdErr := restoreKMS.getPassphrase(f, imageData.csiVolumeHandle)
+						if stdOut != "" {
+							e2elog.Logf("successfully read the passphrase from vault: %s", stdOut)
```

Comment:
this refers to "vault", but that is not the only passphrase storage that cam be configured? Is the error of `getPassphrase()` not clear enough?

Reply:
>this refers to "vault", but that is not the only passphrase storage that cam be configured? Is the error of getPassphrase() not clear enough?

The stdOut is not required here, removed it.